drivers/net/macsec.c

```diff
@@ -1358,6 +1358,24 @@ static struct net_device *get_dev_from_nl(struct net *net,
 	return dev;
 }
 
+static struct macsec_secy *get_secy_from_nl(struct net *net,
```

Comment:
I see no reason on this function. Instead of just adding 3 lines you add 28 lines and changed 12.

Reply:
Okay, will update.